idseq_dag/util/m8.py

```diff
@@ -508,13 +538,15 @@ def any_hits_to_remove(hits):
                         agg_bucket = aggregation.get(agg_key)
                         if not agg_bucket:
                             agg_bucket = {
-                                'count': 0,
+                                'count': 0,  # todo:  rename to nonunique_count -- since count originally meant unique_count
+                                'unique_count': 0,
                                 'sum_percent_identity': 0.0,
                                 'sum_alignment_length': 0.0,
                                 'sum_e_value': 0.0
                             }
                             aggregation[agg_key] = agg_bucket
-                        agg_bucket['count'] += 1
+                        agg_bucket['count'] += get_read_dcr(cdhit_cluster_sizes, read_id)
+                        agg_bucket['unique_count'] += 1
```

Comment:
I missed a few more places to change below, where we average percent_identity the denominator has to be unique_count.